test/functional/lua/runtime_spec.lua

```diff
@@ -138,6 +138,28 @@ describe('runtime:', function()
       exec [[set filetype=new-ft]]
       eq(1, eval('b:lua_ftplugin'))
     end)
+
+    it("'rtp' order is respected", function()
```

Comment:
Can you add a comment saying exactly what the expected behaviour should be, or link to the docs or something?

Reply:
Done